common/src/main/java/com/gregtechceu/gtceu/common/block/CoilBlock.java

```diff
@@ -108,5 +111,15 @@ public String getSerializedName() {
             return name;
         }
 
+        private static final CoilType[] SORTED_BY_TEMPERATURE = Arrays.stream(values())
```

Comment:
what if someone adds a new coil type? you can't go by `values()`.

Reply:
Ah true, hold on...